### ISSUE TITLE
Fail to install with last wiringpi 2.50

### ISSUE THREAD

**Issue by dubocr** [rsg98/wiring-pi#7]
Hi,

Seems there is an issue while installing node-wiring-pi with the latest wiringPi release (2.50).

Here is error logs :
```
nodeWiringPi.target.mk:116: recipe for target 'Release/obj.target/nodeWiringPi/src/wiringPi.o' failed
make: *** [Release/obj.target/nodeWiringPi/src/wiringPi.o] Error 1
make: Leaving directory '/var/lib/homebridge/plugins/homebridge-gpio-device/node_modules/node-wiring-pi/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:189:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.14.98-v7+
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /var/lib/homebridge/plugins/homebridge-gpio-device/node_modules/node-wiring-pi
gyp ERR! node -v v10.15.3
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 
```

Do you known where does it come from ?

Thanks a lot.

**Comment by ClausBendig** [comment]
Have the same issue, with wiringPi 2.46 works.
I installed it with:
`wget https://unicorn.drogon.net/wiringpi-2.46-1.deb && sudo dpkg -i wiringpi-2.46-1.deb`

I think you forgot these upper lines:
```
In file included from ../../nan/nan.h:53:0,
                 from ../src/addon.h:10,
                 from ../src/wiringPi.h:5,
                 from ../src/wiringPi.cc:1:
../src/wiringPi.cc: In function ‘void nodewpi::init_wiringPi(Nan::ADDON_REGISTER_FUNCTION_ARGS_TYPE)’:
../src/wiringPi.cc:689:32: error: ‘PI_MODEL_3’ was not declared in this scope
   NODE_DEFINE_CONSTANT(target, PI_MODEL_3);
                                ^
/home/pi/.node-gyp/8.15.1/include/node/node.h:276:54: note: in definition of macro ‘NODE_DEFINE_CONSTANT’javascript
function fancyAlert(arg) {
  if(arg) {
    $.facebox({div:'#foo'})
  }
}
         v8::Number::New(isolate, static_cast<double>(constant));              \
                                                      ^~~~~~~~
nodeWiringPi.target.mk:116: recipe for target 'Release/obj.target/nodeWiringPi/src/wiringPi.o' failed
```

**Comment by rsg98** [comment]
Should be fixed in 1bb7963e7b615e4a9be27d0a64013dd49d998e7f

(Published as node-wiring-pi npm package 0.0.5)

**Comment by iondarie** [comment]
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 4.19.42-v7+
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /root/node_modules/node-wiring-pi
gyp ERR! node -v v12.4.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 

Node JS 12 install fails, any fix for this?
